### ISSUE TITLE
Intermittent test failures in BrokerBkEnsemblesTests.testSkipCorruptDataLedger()

### ISSUE THREAD

**Issue by merlimat** [apache/pulsar#1388]
https://builds.apache.org/job/pulsar-pull-request/org.apache.pulsar$pulsar-broker/2119/testReport/junit/org.apache.pulsar.broker.service/BrokerBkEnsemblesTests/testSkipCorruptDataLedger/

```
Error Message

Method org.apache.pulsar.broker.service.BrokerBkEnsemblesTests.testSkipCorruptDataLedger() didn't finish within the time-out 6000

Stacktrace

org.testng.internal.thread.ThreadTimeoutException: Method org.apache.pulsar.broker.service.BrokerBkEnsemblesTests.testSkipCorruptDataLedger() didn't finish within the time-out 6000
	at java.util.concurrent.FutureTask.run(FutureTask.java:271)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by merlimat** [comment]
@rdhabalia I'm seeing this multiple times. Would it make sense to increase the timeout from 6seconds to something like 30?

**Comment by ivankelly** [comment]
you need to specify -DtestRetryCount=0 to see the failure

**Comment by sijie** [comment]
This seems to be fixed by #2318
